R/layers.R

```diff
@@ -478,12 +477,7 @@ addRasterImage_SpatRaster <- function(
   options = gridOptions(),
   data = getMapData(map)
 ) {
-  if (!is_installed("terra", "1.6-3")) { # for terra::has.RGB()
-    stop(
-      "`addRasterImage()` for SpatRaster objects requires {terra} 1.6-3 or higher",
-      call. = FALSE
-    )
-  }
+  rlang::check_installed("terra (>= 1.6-3)", reason = "to use addRasterImage() for SpatRaster objects.") # for terra::has.RGB()
```

Comment:
Can we also update https://github.com/rstudio/leaflet/blob/9bf137b8f58d62a199e1eceb906f5b705030b75a/R/normalize-terra.R#L54 ?